packages/storybook/src/stories/ino-fab/ino-fab.stories.js

```diff
@@ -18,74 +18,93 @@ export const DefaultUsage = () => /*html*/ `
      <style>
           ino-fab.customizable-fab {
             --ino-fab-color: ${text(
-              '--ino-fab-color',
-              '#ffffff',
-              'Custom Properties'
-            )};
+  "--ino-fab-color",
+  "#ffffff",
+  "Custom Properties"
+)};
             --ino-fab-background-color: ${text(
-              '--ino-fab-background-color',
-              '#3d40f5',
-              'Custom Properties'
-            )};
+  "--ino-fab-background-color",
+  "#3d40f5",
+  "Custom Properties"
+)};
             --ino-fab-background-color-hover: ${text(
-              '--ino-fab-background-color-hover',
-              '#5d60f7',
-              'Custom Properties'
-            )};
+  "--ino-fab-background-color-hover",
+  "#5d60f7",
+  "Custom Properties"
+)};
             --ino-fab-background-color-active: ${text(
-              '--ino-fab-background-color-active',
-              '#0d10f3',
-              'Custom Properties'
-            )};
+  "--ino-fab-background-color-active",
```

Comment:
This seems to be formatted with the new eslint rules. We should stick to our prettier format until we resolved #283

Reply:
whoops... should be fixed now